MemberExpectations.md

```diff
@@ -42,18 +41,17 @@ professional and respectful manner. Some general guidelines include:
 - Be the model of accountability and leadership. Provide the example of
   ownership and stewardship that everyone can follow to success.
 - Commit to ongoing development and learning best practices for governing.
-- Critique ideas rather than individuals, discussing any concerns in person
-  whenever possible, and taking responsibility for our statements by
-  speaking as much as possible
-  in the first person (.I. statements) rather than in the third person.
+- Criticize ideas rather than people, discussing any concerns in person
+  whenever possible, and taking responsibility for our statements.
   
 While the guidelines above focus primarily on the spaces where
-we participate in official foundation work (github, irc, conferences),
+we participate in official foundation work (GitHub, IRC, conferences),
```

Comment:
Because we have a non-trivial amount of work completed in official meetings of committees and WGs, I'd like to request that be added here. Not sure what the best wording would be, possibly just `meetings` since "official" is stated outside the parenthetical.